### PR TITLE
Feature#43 send email

### PR DESCRIPTION
**Pull request by OchoaHub**
# Description
- Send notifications for covid symptomps and test to users
- Create views for emails (HTML and Text)
- Sending emails
 
 # Testing
  - I've added coverage for testing all new views.
  - You can use the preview of emails that Rails provide to see the routes _password_reset_ and _notification_mailer_

### CODE REVIEW

File: app/mailers/notification_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class NotificationMailer < ApplicationMailer
+    def alert_symp(to)
+      mail(bcc: to, subject: '¡ Alert covid with a shared place !').deliver_now
+    end
+
+    def alert_test(to)
+      mail(bcc: to, subject: '¡ Alert covid with a shared place !').deliver_now
+    end
+end
```
Comment (MiguelOcegueraM) [inline app/mailers/notification_mailer.rb:9]: nice class bro!

**Review by LPMendoza — APPROVED** [review]
It's a good work, but I think you should add screenshots of the mail notifications

File: app/views/password_resets/_new.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="login-box login-page mx-auto w-100" >
   <div class="login-logo">
-    <%= image_tag ("CubreJeta.svg"), :style => "width: 45px" %>
+    <%= image_tag ("Mask.svg"), :style => "width: 45px" %>
```
Comment (jnegrete15) [inline app/views/password_resets/_new.html.erb]: Nice change!!

**Review by MiguelOcegueraM — APPROVED** [review]
nice code my friend

**Review by jnegrete15 — APPROVED** [review]

**Comment by MiguelOcegueraM** [comment]
> nice code my friend

but it could be better jeje

**Pull request merged**